doc/source/metadata/Introduction.rst

```diff
@@ -0,0 +1,723 @@
+    * ``less_than_or_equal_to`` 
+
+Time coordinate variables
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+At present, most parameters have two time coordinate variables, 
+``time`` and ``blend_time``, 
+with a further variable providing the bounds of the time step for parameters 
+where this information is required.
+
+.. literalinclude:: temp12max_prob_ncdump.txt
+    :tab-width: 4
+    :lines: 40-43, 59-64
+
+time
+    The time at which the parameter value is valid.
+
+blend_time
+    Has been added to indicate when the data was processed (blended)
+    to generate this forecast, and can be used to indicate how 'fresh'
+    the data is. 
+    (For the Met Office continually updating dataset, 
+    the ``blend_time`` will not be the same for all validity times,
+    as forecasts in the near future are updated more frequently).
+    Strictly, as ``blend_time`` is not part of the `CF Metadata Conventions`_
+    it should arguably include a 'namespace element' such as ``mosg__``
+    but it doesn't.
+
+.. previously, this included text the text below,
+    but is this still true?
+    There is one special case where blend_time is not set at present:
+    wind direction – more information on this is given later. 
+
+time_bnds
+    Describes the start and end points of the time step
+    (for the maximum temperature example here,
+    it would represent a 12-hour period). 
+    In IMPROVER (as is standard practice for meteorological parameters),
+    the time is at the end of the time step defined by these bounds. 
+
+
+There are two further time coordinate variables which a have been **deprecated**
+and will be removed in the future:
+
+forecast_reference_time
+    This is also a `CF Standard Name`, and used to represent
+    the nominal data time or start time of a model forecast run.
+    However, as IMPROVER generates a blend from multiple sources
+    with different start times, there is no unique data time,
+    so the use of ``blend_time`` is more appropriate.
+
+forecast_period
+    This usually represent the interval between the ``forecast_reference_time``
+    and the validity time (``time``), but as stated above,
+    there is no unique ``forecast_reference_time``, and forecasts valid at
+    different times may have a different ``blend_time``, so at best
+    ``forecast_period`` is unhelpful, at worst it is confusing.
+
+The time coordinate variables share a common set of attributes:
+
+standard_name / long_name
+    A descriptive name, either from the `CF Standard Name`_ list 
+    (e.g. ``time``) or  
+    a non-standard ``long_name`` (e.g. ``blend_time``)
+
+units
+    Units of measure for the quantity.
+    As these are usually in seconds relative to midnight on 1st January 1970,
+    so usually some formatting is required to be human-readable. 
+
+calendar
+    Indicates that a Gregorian (standard) calendar is used. 
+
+bounds
+    Just a pointer to the variable defining the start and end of the time period,
+    if it is present. 
```

Comment:
I think that this could be shortened by writing:
```suggestion
   Pointer to the variable defining the start and end of the time period, if it is present. 
```
however this is only personal preference so I am happy if you prefer to leave it as is. If it is changed, there are a few other instances of sentences beginning "Just a..." which should also be changed.

Reply:
Done - in fact I shortened it slightly further.